ton_lib/src/contracts/methods/get_nft_address_by_index.rs

```diff
@@ -1,17 +1,18 @@
 use crate::block_tlb::TVMStack;
 use crate::contracts::ton_contract::TonContract;
 use crate::error::TLError;
-use crate::tvm_results::GetNftAddressByIndexResult;
+use crate::tvm_results::GetNFTAddressByIndexResult;
 use async_trait::async_trait;
+use num_bigint::BigInt;
 use ton_lib_core::traits::tvm_result::TVMResult;
 
 #[async_trait]
-pub trait GetNftAddressByIndex: TonContract {
-    async fn get_nft_address_by_index(&self, index: i64) -> Result<GetNftAddressByIndexResult, TLError> {
+pub trait GetNFTAddressByIndex: TonContract {
+    async fn get_nft_address_by_index(&self, index: BigInt) -> Result<GetNFTAddressByIndexResult, TLError> {
         let mut stack = TVMStack::default();
-        stack.push_tiny_int(index);
+        stack.push_int(index);
```

Comment:
Does it work with int as well as with tiny_int? We used to use `tiny_int` in old tonlib, but now tests pass as well...

If so, can we change the signature to accept any BigInt-castable type? Like `T: Into<BigInt>`